### ISSUE TITLE
Update for upcoming changes to the OSATE graphical editor API changes

### ISSUE THREAD

**Issue by philip-alldredge** [loonwerks/AGREE#22]
Sorry for using the issue tracker for a question. I wasn't sure who to contact.

We are currently looking at making breaking updates to the OSATE graphical editor's API. (https://github.com/osate/osate2/issues/2336) The AGREE graphical editor plugin and the AGREE simulator both use the APIs and one or both may be affected by this. What is the current stance on accepting pull requests and does AGREE target a specific OSATE version or does it track the latest nightly release?

**Comment by kfhoech** [comment]
Currently, due to personnel commitments to multiple deliverables other programs we are behind on keeping AGREE up to date.  And, as you might see, the issue list is accumulating. However, we do have hope that we'll be able to find the resources to get up to date.

Previously, when OSATE did not practice semantic versioning, we needed to lock AGREE to a particular version. Now that OSATE is using semantic versioning, we are planning to do the same and track the latest developments.  We'll probably add a develop branch as we had had before when we do that.

Certainly, yes, we are interested in accepting pull requests. Just be aware that we are limited in personnel availability due to the other commitments and COVID-19-related furloughs, and it may be a while before we can catch up and merge the pull request.

Thanks for your help in keeping AGREE relevant and useful to the community!

**Comment by philip-alldredge** [comment]
FYI. I have opened a pull request which will fix the graphical editor and simulator. The API changes are planned to be merged into the OSATE graphical editor during the OSATE 2.8.1 development cycle. They are *not* currently in the OSATE master branch. See https://github.com/osate/osate2/issues/2336

**Comment by kfhoech** [comment]
Resolved by #25
